### PR TITLE
Add configuration for github

### PR DESCRIPTION
**Pull request by y6vmeq**
fixed issue #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,6 +202,17 @@ Requirements
      default: `https://localhost`
   - `jenkins2_gitlab_token_cred` - gitlab token ID (from credentials: gitlabtoken in example below)   
      default: `gitlab_token`
+# Github configuration
+  - `jenkins2_github_enabled` - to enable github config   
+     default: `False`
```
Comment (ViachaslauKabak) [inline README.md]:
```suggestion
     default: `false`
```

File: README.md [hunk 2]
```diff
@@ -202,6 +202,17 @@ Requirements
...
+  - `github_external_url` - github external url   
+     default: `https://localhost`
+  - `github_manage_hooks` - manage github hooks   
+      default: `True`
```
Comment (ViachaslauKabak) [inline README.md]:
```suggestion
      default: `true`
```

File: defaults/main.yml [hunk 3]
```diff
@@ -268,6 +273,13 @@ jenkins2_gitlab_enabled: false
 gitlab_external_url: "https://localhost"
 jenkins2_gitlab_token_cred: "gitlab_token"
 
+# Setup github connection
+jenkins2_github_enabled: False
```
Comment (ViachaslauKabak) [inline defaults/main.yml]:
```suggestion
jenkins2_github_enabled: false
```

File: defaults/main.yml [hunk 4]
```diff
@@ -268,6 +273,13 @@ jenkins2_gitlab_enabled: false
...
+github_external_url: "https://api.github.com"
+github_manage_hooks: "true"
```
Comment (ViachaslauKabak) [inline defaults/main.yml]:
```suggestion
github_manage_hooks: true
```

File: defaults/main.yml [hunk 5]
```diff
@@ -268,6 +273,13 @@ jenkins2_gitlab_enabled: false
...
+github_client_cache_size: 20
+jenkins2_github_token_cred: "github_token"
```
Comment (ViachaslauKabak) [inline defaults/main.yml]:
```suggestion
jenkins2_github_token_cred: github_token
```

File: tasks/configure_github.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+- name: "Configure Github"
```
Comment (ViachaslauKabak) [inline tasks/configure_github.yml]:
```suggestion
- name: Configure Github
```

File: tasks/configure_github.yml [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+  jenkins_script:
+    url: "http://localhost:{{ jenkins2_config_http_port }}"
```
Comment (ViachaslauKabak) [inline tasks/configure_github.yml]:
```suggestion
    url: 'http://localhost:{{ jenkins2_config_http_port }}'
```

File: tasks/configure_github.yml [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+    script: "{{ lookup('template', 'set-github-connection.groovy.j2') }}"
```
Comment (ViachaslauKabak) [inline tasks/configure_github.yml]:
```suggestion
    script: '{{ lookup('template', 'set-github-connection.groovy.j2') }}'
```

Comment (y6vmeq) [reply tasks/configure_github.yml]: Not YAML syntax.

File: defaults/main.yml (hunk 4 above)
Comment (y6vmeq) [reply defaults/main.yml]: Must be passed as a string type

**Pull request merged**